src/quo/components/pin_input/view.cljs

```diff
@@ -0,0 +1,27 @@
+(ns quo.components.pin-input.view
```

Comment:
Because the `pin-input.view` component is now part of the Quo public API, it should have component tests.

Reply:
do we know why? `now part of the Quo public API` sounds strange as a reason. This component is too simple, to have a test for it

Reply:
The reason is actually very commonly used @flexsurfer. It's one of the main reasons library authors use to assess what needs to be tested. If it's public for consumption, it should probably be tested.

The test is also simple & quick to write if you are used to it. The basic coverage can automatically catch common mistakes, like invalid hiccup, incorrectly used default values, incorrect number of pins generated. I know it's simple for you, but tests are all about the next person to touch the code, not necessarily the original author.

But hey, I'm just sharing the guideline because it's unanimously respected for new components, so it's best if we follow here as well.